### ISSUE TITLE
Dataset Build?

### ISSUE THREAD

**Issue by AnirudhDagar** [garythung/trashnet#2]
Hi!
Great work with the model and the idea.

I'm working on something very very similar and have a few questions.
I can understand due to the limited availability of the data, the CNN models don't perform as expected. If you can explain the dataset retrieval process, it would be very helpful.
How did you get the dataset? The README suggests, all of it was manually built. If yes then did you try some data mining or scraping other than the manual process?

PS. The images look to be captured in very specific conditions like a white cardboard background etc.
How do you see this translate into a real-world scenario where a similar model is deployed using a camera??

**Comment by gale31** [comment]
Hi Anirudh!

Thank you! The dataset is still a 'baby' -- I have taken photos of trash for only about three days in total... The photographs are taken by my camera, and in my town -- for a few days in a row, I woke up early in the morning, and went to the Black sea beaches in my town, where there is plenty of trash. I also randomly took pictures of trash on the street or foresty environments. All of the photographs were taken in real-world environments in my town. It is something like a personal volunteering project, and hopefully I would keep growing this dataset so machine learning models can successfully be applied to it.

Hope this clears (get the pun?) things up. Please reach out if you have any more questions. It makes me super happy that someone can already find this dataset useful (and I started it just a week ago! :). I would also be interested in your idea if/whenever you are ready to share it. 

All the best :)

**Comment by AnirudhDagar** [comment]
```
Hi Anirudh!

Thank you! The dataset is still a 'baby' -- I have taken photos of trash for only about
three days in total... The photographs are taken by my camera, and in my town -- 
for a few days in a row, I woke up early in the morning, and went to 
the Black sea beaches in my town, where there is plenty of trash.
I also randomly took pictures of trash on the street or foresty environments.
All of the photographs were taken in real-world environments in my town.
It is something like a personal volunteering project, and hopefully I would keep
growing this dataset so machine learning models can successfully be applied to it.

Hope this clears (get the pun?) things up. Please reach out if you have any more questions.
It makes me super happy that someone can already find this dataset useful
(and I started it just a week ago! :).
I would also be interested in your idea if/whenever you are ready to share it.

All the best :)
```

Hi Gale!
Replying to your email here, regarding gale31/TIDY [(Your Repo)](https://github.com/gale31/TIDY) here in this issue. It is very nice to know that other people are working on this problem. We at [Data Science Group, IIT R](https://github.com/dsgiitr) are working to develop a prototype dustbin that would classify waste and segregate that based on simple Computer Vision and Machine Learning models. The only problem we are facing is the dataset itself. We have some plans in mind and have already started work on both the fronts, Software+Hardware

**Comment by OricTosh** [comment]
Also Interested in using this dataset for detection of litter/rubbish/trash and my project is looking to do something with AI and robots. 
During my research I found an app which a person downloads, set-up a profile, then they photograph a bit of litter, add a tag. The photo is uploaded to the app owner AWS storage cloud. 
There site is [here](https://www.litterati.org/) 
I have tried to contact them, requesting access to their dataset, yet no reply 
These photos taken would be good real life instances of litter 
Chris Hills
